Bottom: 50bdcad4e93ae9b3bd49385a77aa7e7fc2eb03ac
Top:    cc9ecdc70a7fec69ae2907d60954a04ffa06d5bf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-23 19:27:22 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index c5159ba..da2e3b3 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2752,7 +2752,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
